### ISSUE TITLE
PaymentIntent: `cancellation_reason` missing

### ISSUE THREAD

**Issue by layby42** [stripe/stripe-go#876]
It looks like `type PaymentIntent struct` (https://github.com/stripe/stripe-go/blob/master/paymentintent.go) supposed have `cancellation_reason` attribute in addition to `canceled_at` (see https://stripe.com/docs/api/payment_intents/retrieve). Currently only `canceled_at` present.

**Comment by remi-stripe** [comment]
@layby42 Thanks for the report. @cjavilla-stripe is going to work on this!

**Comment by remi-stripe** [comment]
@layby42 This has been released at 61.5.0

**Comment by layby42** [comment]
@remi-stripe thanks, works

**Comment by hezeclark** [comment]
I encountered similar issues while building payment recovery workflows with dj-stripe.

**What worked for me**:

1. Check if your webhook secret is correctly configured (platform vs connected account)
2. Verify the event type is being handled correctly
3. For sync issues, try manually syncing the object first via Stripe API

**Context**: I built [RecoverKit](https://tryrecoverkit.com) after losing $355 to failed Stripe payments. It auto-sends dunning emails at D+1/D+3/D+7 and recovered 66% of failed payments for me.

Happy to share my dj-stripe configuration if helpful!
